tests/lite/test_wrappers.py

```diff
@@ -45,9 +45,9 @@ def test_lite_module_wraps():
         (16, torch.float32, torch.float16),
         (16, torch.float64, torch.float16),
         (16, torch.long, torch.long),
-        pytest.param("bf16", torch.float32, torch.bfloat16, marks=RunIf(min_torch="1.10")),
-        pytest.param("bf16", torch.float64, torch.bfloat16, marks=RunIf(min_torch="1.10")),
-        pytest.param("bf16", torch.bool, torch.bool, marks=RunIf(min_torch="1.10")),
+        pytest.param("bf16", torch.float32, torch.bfloat16, marks=RunIf(min_torch="1.10", bf16_cuda=True)),
+        pytest.param("bf16", torch.float64, torch.bfloat16, marks=RunIf(min_torch="1.10", bf16_cuda=True)),
+        pytest.param("bf16", torch.bool, torch.bool, marks=RunIf(min_torch="1.10", bf16_cuda=True)),
```

Comment:
Ah right, our CI GPU doesn't support bf16 :) 

I checked that it does pass on the 3090. Looks good! Thx

Reply:
Thanks for your verification!